File: atividade_6/invertida.c
#include <stdio.h>
#include <string.h>

int main() {
    char frase1[1000], frase2[1000];
    printf("Digite a primeira frase: ");
    fgets(frase1, sizeof(frase1), stdin);
    printf("Digite a segunda frase: ");
    fgets(frase2, sizeof(frase2), stdin);

    printf("Frase 1 invertida com A trocado por *: ");
    for (int i = strlen(frase1) - 1; i >= 0; i--) {
        if (frase1[i] == 'A' || frase1[i] == 'a') {
            putchar('*');
        } else {
            putchar(frase1[i]);
        }
    }

    printf("\nFrase 2 invertida com A trocado por *: ");
    for (int i = strlen(frase2) - 1; i >= 0; i--) {
        if (frase2[i] == 'A' || frase2[i] == 'a') {
            putchar('*');
        } else {
            putchar(frase2[i]);
        }
    }

    return 0;
}

File: atividade_6/invertida.py
frase1 = input("Digite a primeira frase: ")
frase2 = input("Digite a segunda frase: ")

def inverter_e_substituir(frase):
    return ''.join(['*' if letra in 'Aa' else letra for letra in reversed(frase)])

frase1_invertida = inverter_e_substituir(frase1)
frase2_invertida = inverter_e_substituir(frase2)

print(f"Frase 1 invertida com A trocado por *: {frase1_invertida}")
print(f"Frase 2 invertida com A trocado por *: {frase2_invertida}")
